hooks/useRealm.tsx

```diff
@@ -258,7 +265,8 @@ export default function useRealm() {
     ownDelegateCouncilTokenRecords,
     config,
     currentPluginPk,
-    isLockTokensMode,
+    isDefaultVsrMode,
```

Comment:
how annoying would it be to move `isDefaultVsrMode` and `isHeliumVsrMode` to their own hooks? I don't know who invented the mega-hook but I want to abolish it.

Reply:
What would it be, useVsrMode? that just returns a enum.

Reply:
yeah sounds good (though don't use an actual enum, just a literal union `"helium" | "default" | undefined`)